domain-management/src/main/java/org/jboss/as/domain/management/security/SecurityRealmResourceDefinition.java

```diff
@@ -48,6 +50,17 @@
  */
 public class SecurityRealmResourceDefinition extends SimpleResourceDefinition {
 
+    static {
+        //register the Openssl Provider, if possible
+        //not really sure if this is the best place for it
```

Comment:
Without this OpenSSL is not usable, so there is not much point in merging it. IMHO it is better to just merge this and then it can be removed why Elytron is ready for it.